status/deploy.yaml

```diff
@@ -0,0 +1,118 @@
+---
+version: "2.0"
+
+endpoints: 
+  nodeendpoint-pg:
+    kind: ip
+
+services:
+  node:
+    image: harbor.status.im/wakuorg/nwaku:v0.31.0
+    expose:
+      - port: 60000
+        as: 60000
+        proto: tcp
+        to:
+          - global: true
+            ip: nodeendpoint-pg
+
+      - port: 8003
+        as: 8003
+        proto: tcp
+        to:
+          - global: true
+            ip: nodeendpoint-pg
+
+      - port: 9005
+        proto: udp
+        to:
+          - global: true
+            ip: nodeendpoint-pg
+    env:
+#Mandatory
+      - IP_ADDR=
+#Optional
+      - NODEKEY=
+
+    command:
+      - sh
+      - '-c'
+    args:
+      - >
+        if [ -z "${IP_ADDR}" ]; then
```

Comment:
I tested this deployment again with the Europlots provider. I did not receive the errors you indicated.
As for the method you suggested:
```
IP_ADDR=$(curl api.ipify.org)
echo $IP_ADDR
```
it is not relevant, since it does not correctly determine the external IP issued by the provider on a static port.
![image](https://github.com/user-attachments/assets/67207bb7-f501-40b9-8a13-adc15ea2de9b)
![image](https://github.com/user-attachments/assets/9a9d8ae3-adbb-4622-b4c5-68bfa726eb71)
[17771530-1-1-logs-2024-08-25.txt](https://github.com/user-attachments/files/16741759/17771530-1-1-logs-2024-08-25.txt)

Reply:
Yeah, if you read the docs, the problem is that egress traffic goes through provider egress controler, but ingress goes through your IP, so that is why you cannot use ipify - I tried tha tin the past:) I would love for it to be possible, but sadly it is not.

Reply:
There is no problem without a solution. Try again with :
```
METALLB_IP=$(for src in ifconfig.me icanhazip.com checkip.amazonaws.com; do timeout 5 wget -qO- "$src" && break; done) && PROVIDER_IP=$(timeout 5 getent hosts "$AKASH_CLUSTER_PUBLIC_HOSTNAME" | awk '{print $1}') && echo "Pod Assigned IP: $METALLB_IP Provider IP: $PROVIDER_IP"
IP_ADDR=$METALLB_IP
```

Reply:
Works great!
![image](https://github.com/user-attachments/assets/e4e2c75e-f59b-4234-87a7-5f555061ed66)

Reply:
```
    args:
      - |-
        METALLB_IP=$(for src in ifconfig.me icanhazip.com checkip.amazonaws.com; do timeout 5 wget -qO- "$src" && break; done) && PROVIDER_IP=$(timeout 5 getent hosts "$AKASH_CLUSTER_PUBLIC_HOSTNAME" | awk '{print $1}') && echo "Pod Assigned IP: $METALLB_IP Provider IP: $PROVIDER_IP"
        IP_ADDR=$METALLB_IP
        if [ "${NODEKEY}" != "" ]; then
            NODEKEY=--nodekey=${NODEKEY}
        fi
```